synthetic_data/distinct_generators/int_generator.py

```diff
@@ -20,4 +20,6 @@ def random_integers(
 
     :return: np array of integers
     """
+    if max_value <= 0:
```

Comment:
There is this edge case where the max_value in the report is 0. You cannot set 0 as the max_value in `rng.integers`, so I set this to the default max_value